nhsn/delphi_nhsn/pull.py

```diff
@@ -52,7 +88,11 @@ def pull_nhsn_data(socrata_token: str, backup_dir: str, custom_run: bool, logger
         Dataframe as described above.
     """
     # Pull data from Socrata API
-    df = pull_data(socrata_token, dataset_id="ua7e-t2fy")
+    df = (
+        pull_data(socrata_token, dataset_id="ua7e-t2fy")
```

Comment:
It feels like the dataset_id should be in a constants file somewhere, like in google_symptoms